keyboards/pimoroni/keybow2040/keybow2040.c

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+#include "rgb_matrix.h"
+
+void keyboard_post_init_user(void) {
+	debug_enable = true;
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+#ifdef CONSOLE_ENABLE
+	dprintf("KL: kc: 0x%04X, col: %2u, row: %2u, "
+		"pressed: %u, time: %5u, "
+		"int: %u, count: %u\n",
+		keycode, record->event.key.col, record->event.key.row,
+		record->event.pressed, record->event.time,
+		record->tap.interrupted, record->tap.count);
+#endif
+  return true;
+}
+
```

Comment:
The `*_user` functions must not be defined at the keyboard level, and this debugging code probably should be removed from the final version anyway.
```suggestion
```

Reply:
Absolutely: this is like the `eeprom_transient` setting.

I realise I forgot to mention in the PR description that I deliberately left in some debugging setup. (Note to self: mentioning it in the commit messages is not enough.)  Oops, sorry for wasting your time!

Reply:
I don't even try to read commit messages in PRs to `qmk_firmware`, because at the merge time the PR will get squashed to a single commit anyway, and therefore there is no requirement to keep the individual commit messages in PRs clean.

Reply:
Gone in round two; I've pre-squashed the PR to one commit as well